npm/npm_import.bzl

```diff
@@ -355,6 +356,13 @@ def npm_translate_lock(
 
         warn_on_unqualified_tarball_url: Deprecated. Will be removed in next major release.
 
+        npm_package_target_name: The name of linked `npm_package` targets. When `npm_package` targets are linked as pnpm workspace
+            packages, the name of the target must align with this value.
+
+            By default the directory name of the target is used.
```

Comment:
Add a mention that `{dirname}` is replaced by the current directory name and can be included in a pattern such as `{dirname}_pkg`